ANDROID.md

```diff
@@ -0,0 +1,87 @@
+## install MAVProxy on Android ?
+
+Yes, its possible to install MAVProxy on Android, toally unsupported, no guaranetees, good luck, but here goes.
+
+### install/load Termux Android app from .apk on github
+see https://github.com/termux/termux-app for more.
+
+eg: https://github.com/termux/termux-app/releases/download/v0.118.0/termux-app_v0.118.0+github-debug_armeabi-v7a.apk 
+
+tip: do NOT install from play store, its a really old version. 
+
+Once onstalled, in the termux console run these commands:
+```
+pkg upgrade 
+```
+[ answer 'y' to all questions.]
+
+```
+pkg install python
+```
+[3.10.x at time of testing]
+
+```
+pkg install libxml2* 
+```
+[ for -dev pkg ]
+
+```
+pkg install libxslt*
+```
+[ for -dev pkg ]
+
+```
+pkg install python-*   
+```
+[ for python-dev , be sure to include the '-' here.]
+
+```
+pip install --upgrade pip
+```
+pip is needed for the next few packages
+
+```
+pip install wheel
+pip install pymavlink
+```
+wheel isn't installed automatically in some cases, so we install it manually first
+
+#https://superuser.com/questions/1724176/why-pip-throwing-an-error-when-installing-numpy-in-termux
+```
+MATHLIB="m" pip3 install numpy
+```
+numpy doesnt install without this extra prefix on android , so we do it first as well.
+
+```
+pip install mavproxy
+```
+finally we install mavproxy!
+
+```
+pkg install mlocate
+updatedb
+```
+optionally other things that might be helpful
+
+```
+locate mavproxy.py
+```
+where is it?
+
+```
+mavproxy.py
+
+mavproxy.py --master=udp:1.2.3.4:14550
+```
+a few different ways to run it!
+eg: if u have udp mavlink data being forward to your phone over wifi, then 1.2.3.4 should be your PHONE's ip address, and it will recieve the data.
```

Comment:
```suggestion
eg: if u have udp mavlink data being forward to your phone over wifi, then 1.2.3.4 should be your PHONE's ip address, and it will receive the data.
```